user-guide/Standard_Apps/IDP/Tutorials/IDP_Tutorial_DiscoveryAndProvisioning.md

```diff
@@ -0,0 +1,196 @@
+---
+uid: IDP_Tutorial_DiscoveryAndProvisioning
+keywords: idp tutorial, idp kata
+---
+
+# Getting started with IDP and onboarding new equipment
+
+This tutorial will show you how to install IDP and onboard new equipment. By way of example, the Microsoft server running DataMiner will be onboarded, as this is easily accessible equipment.
+
+Expected duration: 20 minutes.
+
+<!-- TODO: Uncomment and fill in with correct info when Kata is released.
+> [!TIP]
+> See also: [Kata #xx: IDP Introduction](https://community.dataminer.services/courses/kata-xx/)
+-->
+
+## Prerequisites
+
+The DataMiner System used for this tutorial has to meet the following requirements:
+
+- DataMiner is installed on a machine running a Windows Server version (e.g. Windows Server 2022).
+- The DataMiner System is [connected to dataminer.services](xref:Connecting_your_DataMiner_System_to_the_cloud).
+- The DataMiner version is 10.3.0 [CU0] or higher.
+
+> [!IMPORTANT]
+> If you are using a DataMiner version lower than 10.4.0 or 10.4.3, install the [IDP Migration](https://community.dataminer.services/download/idp-migration/) package before you start the tutorial.
+
+> [!NOTE]
+> If you use a [DaaS system](xref:Creating_a_DMS_in_the_cloud), these prerequisites are automatically met.
+
+## Overview
+
+- [Step 1: Deploy IDP from the DataMiner Catalog](#step-1-deploy-idp-from-the-dataminer-catalog)
+- [Step 2: Configure the discovery profile](#step-2-configure-the-discovery-profile)
+- [Step 3: Configure the scan range](#step-3-configure-the-scan-range)
+- [Step 4: Run the discovery](#step-4-run-the-discovery)
+- [Step 5: Provision the discovered element(s)](#step-5-provision-the-discovered-elements)
+
+## Step 1: Deploy IDP from the DataMiner Catalog
+
+1. [Deploy the IDP package from the Catalog](xref:Installing_DataMiner_IDP#deploying-the-idp-package).
+
+1. Connect to your DataMiner System in DataMiner Cube.
+
+You will notice that IDP will be automatically installed and set up for you.
+
+## Step 2: Configure the discovery profile
+
+Discovery profiles define which information should be retrieved from the device. Some devices require authentication, which means credentials have to be filled in in the discovery profile. This is also the case here.
+
+1. Open the IDP app.
+
+1. Go to *Admin* > *Discovery*.
+
+1. On the right, in the *Discovery Profiles* table, select the row with the name `Default - Microsoft Platform`.
+
+1. Click the *Edit* button.
+
+1. Fill in the *Credentials* settings.
+
+   - If you are using a DaaS system, use the default username and password that were configured during the creation of the DaaS system.
```

Comment:
If I read the [DaaS help](https://docs.dataminer.services/user-guide/Getting_started/DaaS/Creating_a_DMS_on_dataminer_services.html) correctly, the username doesn't necessarily have to be Admin. If this is indeed the case, to avoid confusion it might be better to leave out line 63-65 and instead add a link to that DaaS creation page here for reference.

Reply:
Removed step 2 fully as it wasn't needed. I'll briefly discuss in during the Kata itself, but isn't needed for the tutorial.